index.html

```diff
@@ -199,7 +199,21 @@ <h2 class="subt">Skills</h2>
           </div>
         </div>
       </section>
-    </main>
-    <footer></footer>
+      <footer class="contact-form">
+        <section class="form-section">
+          <div class="contact-me">
+            <h2 >Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="https://formspree.io/f/xrgjlnej" method="POST" class="form">
+            <input type="text" class="text" value="Yeremias" name="fullname" maxlength="30" required>
+            <input type="email" class="text" value="natajayanj14@gmail.com" name="email" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$">
+            <textarea class="textarea" placeholder="Write your message here" maxlength="500" required></textarea>
```

Comment:
- [ ] [OPTIONAL] please don't use a `value` in the name or email input this should be replaced by `placeholder`. 
- [ ] [OPTIONAL] Additionally, the name that was put in the template is just as an example the placeholder should say something like "Full Name" in the name `input` and "Email Address" in the email `input`